src/Tribe/Template/Month.php

```diff
@@ -288,6 +279,40 @@ private static function view_more_link( $date ) {
 			return esc_url_raw( $day_link );
 		}
 
+
+		/**
+		 * Get all the events in the month by directly querying the postmeta table
+		 * Also caches the postmeta and terms for the found events
+		 */
+		protected function set_events_in_month() {
+			global $wpdb;
+
+			// note: this query returns the event dates as timestamps, makes date comparison faster later
+			$events_request =
+				$wpdb->prepare(
+					"SELECT tribe_event_start.post_id as ID,
+							UNIX_TIMESTAMP(tribe_event_start.meta_value) as EventStartDate,
+							UNIX_TIMESTAMP( tribe_event_end_date.meta_value) as EventEndDate
+					FROM $wpdb->postmeta AS tribe_event_start
```

Comment:
Nice. I just checked out your branch and tested it and your solution here works swimmingly! :+1: